test/compress/issue-208.js

```diff
@@ -30,25 +30,37 @@ do_update_rhs: {
 
 mixed: {
     options = {
-        global_defs: { DEBUG: 0 }
+        global_defs: {
+            DEBUG: 0,
+            ENV: 1,
+            FOO: 2,
+        }
     }
     input: {
+        const ENV = 3;
+        var FOO = 4;
+        f(ENV);
+        --FOO;
         DEBUG = 1;
         DEBUG++;
         DEBUG += 1;
         f(DEBUG);
         x = DEBUG;
     }
     expect: {
+        const ENV = 3;
+        var FOO = 4;
```

Comment:
overriding `var FOO` should warn. 

One could argue in this specific case if it is set to the same value (4) it shouldn't - but that's too much work.